app/timetables/admin.py

```diff
@@ -131,6 +131,22 @@ class MenuItemAdmin(TimeStampAdmin):
         }),
     ]
 
+    search_fields = (
+        'timetable__name',
+        'cycle_day',
+        'meal__name',
+        'course__name',
+        'dish__name'
+        )
+
+    list_filter = (
+        'timetable',
+        'cycle_day',
+        'meal',
+        'course',
+        'dish'
+        )
```

Comment:
On a closer look, I think the list_filter needs to have the `__` to the lookup field for those foreign key fields with verbose_name kwarg added to the lookup field in the model. Just like you did [here](https://github.com/teamtaverna/core/pull/291/files) in another PR.

Reply:
Ok, I will add the verbose_name kwarg on name fields in models, and lookup for it in the list_filter